File: output/ChatGPT/avatar/Java/Python/codeforces_412_A.py
n, k = map(int, input().split())
a = input()

if k > n//2:
    while k < n:
        print("RIGHT")
        k += 1
else:
    while k > 1:
        print("LEFT")
        k -= 1

if k == 1:
    for i in range(len(a)):
        print("PRINT", a[i])
        if i+1 < len(a):
            print("RIGHT")
else:
    for i in range(len(a)-1, -1, -1):
        print("PRINT", a[i])
        if i-1 >= 0:
            print("LEFT")

File: output/CodeLlama/avatar/Python/Java/codeforces_412_A.java
import java.util.Scanner;

public class codeforces_412_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int k = sc.nextInt();
        String v = sc.next();
        String[] d = {"LEFT", "RIGHT"};
        boolean f = k - 1 < n - k;
        int m = Math.min(k - 1, n - k);
        String[] a = new String[m + 2 * f + v.length()];
        for (int i = 0; i < m; i++) {
            a[i] = d[f ? 1 : 0];
        }
        for (int i = 0; i < v.length(); i++) {
            a[i + m] = "PRINT " + v.charAt(i);
            a[i + m + v.length()] = d[f ? 0 : 1];
        }
        for (int i = 0; i < a.length - 1; i++) {
            System.out.println(a[i]);
        }
    }
}